### PR TITLE
Feature #8791: Tile inspector improvements

### PR DESCRIPTION
**Pull request by joshtucker132**
Feature #8791: Improve Tile Inspector

- Added broken flag to path group box.
- Added block brake and indestructible flags to track group box.
- Removed broken flag from columns.

We believe we have resolved all the functionality issues we were having. However, we had some git difficulties so all commits dated before Apr, 26, 2019 should not be included in the pull request.

### CODE REVIEW

File: data/language/en-GB.txt [hunk 1]
```diff
@@ -3754,6 +3754,9 @@ STR_6303    :Downloading object ({COMMA16} / {COMMA16}): [{STRING}]
 STR_6304    :Open scenery picker
 STR_6305    :Multithreading
 STR_6306    :{SMALLFONT}{BLACK}Experimental option to use multiple threads to render, may cause instability.
+STR_6307    :{WINDOW_COLOUR_2}Block Brake Closed
```
Comment (duncanspumpkin) [inline data/language/en-GB.txt]: Remove all but these changes to en-GB.

**Comment by duncanspumpkin** [comment]
How did you accidentally end up deleteing Localisation.cpp?

**Comment by duncanspumpkin** [comment]
As this will be modifiying a game command you will need to increment the network version (network.cpp). The commit message is still not quite right either it should be 

```
Feature #8791: Improve Tile Inspector

Added broken flag to path group box. Removed broken flag from column.
Added ... 
```
The most important thing is the `#` as that is what links it in github to an issue.

**Comment by duncanspumpkin** [comment]
You've also accidentally added a shortcut to the repo. Please delete that.

**Comment by joshtucker132** [comment]
So change the network stream version from 19 to 20?

Also, I re-added the localisation.cpp file. I believe it was accidentally deleted due to local merge conflicts. However, it appears that for some reason it thinks changes were made even though my local file matches the one in the main repository.

**Comment by joshtucker132** [comment]
Also how do I fix the shortcut to the repo?

**Comment by Broxzier** [comment]
Hey @joshtucker132, there's many commits in here that aren't part of the changes you made. Would you like some help cleaning up this PR?

It looks like the only commit you want to keep is dc8d335faea864b81565cc31cc0e7a461a7c1164, with 6c7935fce56f2258dd7b09a864722c383c87fefa squashed to fix the deletion of `Localization.cpp`.

File: src/openrct2/world/TileInspector.h [hunk 2]
```diff
@@ -48,6 +49,8 @@ enum TILE_INSPECTOR_INSTRUCTION_TYPE
     TILE_INSPECTOR_SCENERY_SET_QUARTER_COLLISION,
     TILE_INSPECTOR_BANNER_TOGGLE_BLOCKING_EDGE,
     TILE_INSPECTOR_CORRUPT_CLAMP,
+    TILE_INSPECTOR_TRACK_SET_BLOCK_BRAKE,
+    TILE_INSPECTOR_TRACK_SET_INDESTRUCTIBLE
```
Comment (Broxzier) [inline src/openrct2/world/TileInspector.h]: Please add a trailing comma here.

**Comment by duncanspumpkin** [comment]
Note: Squash merge this to clean up history.

File: src/openrct2-ui/windows/TileInspector.cpp [hunk 3]
```diff
@@ -312,14 +313,16 @@ static rct_widget PathWidgets[] = {
...
-#define TRA_GBPT (TRA_GBPB + 16 + 3 * 21)       // Track group box properties top
+#define TRA_GBPT (TRA_GBPB + 16 + 5 * 21)       // Track group box properties top
 #define TRA_GBDB (TRA_GBPT + GROUPBOX_PADDING)  // Track group box info bottom
 #define TRA_GBDT (TRA_GBDB + 20 + 7 * 11)       // Track group box info top
 static rct_widget TrackWidgets[] = {
     MAIN_TILE_INSPECTOR_WIDGETS,
     { WWT_CHECKBOX,         1,  GBBF(WH - TRA_GBPT, 0, 0),  STR_TILE_INSPECTOR_TRACK_ENTIRE_TRACK_PIECE,    STR_NONE }, // WIDX_TRACK_CHECK_APPLY_TO_ALL
       SPINNER_WIDGETS      (1,  GBBL(1), GBBR(1), GBBT(WH - TRA_GBPT, 1) + 3, GBBB(WH - TRA_GBPT, 1) - 3,   STR_NONE, STR_NONE),  // WIDX_TRACK_SPINNER_HEIGHT{,_INCREASE,_DECREASE}
     { WWT_CHECKBOX,         1,  GBBF(WH - TRA_GBPT, 0, 2),  STR_TILE_INSPECTOR_TRACK_CHAIN_LIFT,            STR_NONE }, // WIDX_TRACK_CHECK_CHAIN_LIFT
+    { WWT_CHECKBOX,         1,  GBBF(WH - TRA_GBPT, 0, 1),  STR_TILE_INSPECTOR_TRACK_BLOCK_BRAKE,           STR_NONE }, // WIDX_PATH_CHECK_BLOCK_BRAKE_CLOSED
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/TileInspector.cpp]: Row 3 `GBBF(WH - TRA_GBPT, 0, 3)`

File: src/openrct2-ui/windows/TileInspector.cpp [hunk 4]
```diff
@@ -312,14 +313,16 @@ static rct_widget PathWidgets[] = {
...
+    { WWT_CHECKBOX,         1,  GBBF(WH - TRA_GBPT, 0, 1),  STR_TILE_INSPECTOR_TRACK_IS_INDESTRUCTIBLE,     STR_NONE }, // WIDX_PATH_CHECK_IS_INDESTRUCTIBLE
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/TileInspector.cpp]: Row 4 `GBBF(WH - TRA_GBPT, 0, 4)`

File: data/language/en-GB.txt [hunk 5]
```diff
@@ -3766,6 +3766,9 @@ STR_6315    :{WINDOW_COLOUR_2}Pathfind Goal: {BLACK}{INT32}, {INT32}, {INT32} di
 STR_6316    :{WINDOW_COLOUR_2}Pathfind history:
 STR_6317    :{BLACK}{INT32}, {INT32}, {INT32} dir {INT32}
 STR_6318    :Network desync detected.{NEWLINE}Log file: {STRING}
+STR_6319    :{WINDOW_COLOUR_2}Block Brake Closed
+STR_6320    :{WINDOW_COLOUR_2}Indestructible
+STR_6321    :Broken
```
Comment (Gymnasiast) [inline data/language/en-GB.txt]: The path is not broken, the path addition is.

Also, this lacks the `{WINDOW_COLOUR_2}` formatting code.

```suggestion
STR_6321    :{WINDOW_COLOUR_2}Addition is broken
```

**Pull request merged**